es/index.mjs

```diff
@@ -75,7 +75,7 @@ function noDelay() {
  * @return {number} - delay in milliseconds
  */
 export function exponentialDelay(retryNumber = 0) {
-  const delay = Math.pow(2, retryNumber) * 100;
+  const delay = Math.pow(2, retryNumber) * 1000;
   const randomSum = delay * 0.2 * Math.random(); // 0-20% of the delay
   return delay + randomSum;
 }
```

Comment:
Suggesting some updates to satisfy the latest [Google recommendations for exponential backoff](https://cloud.google.com/storage/docs/retry-strategy#exponential-backoff). 

```ts
export function exponentialDelay(retryNumber = 0) {
  const randomMs = Math.random() * 1000; // 0 to 1000 milliseconds
  const exponentialDelay = Math.pow(2, retryNumber) * 1000 + randomMs;
  const maxBackoff = 64 * 1000; // 64 seconds
  const delay = Math.min(exponentialDelay, maxBackoff);
  const deadline = 10 * 60 * 1000; // 10 minutes
  if (delay > deadline) {
    throw new Error(
      `Exponential retry delay exhausted after ${
        deadline / 1000 / 60
      } minutes ${retryNumber} retries.`,
    );
  }
  return delay;
}
```

We'd probably want to allow the user to configure the `maxBackoff` and `deadline` vars to their liking.